framework/src/engine/consensus/consensus.ts

```diff
@@ -452,6 +462,16 @@ export class Consensus {
 					},
 					'Discarding block due to double forging',
 				);
+				this._logger.info(
```

Comment:
(Optional) That would look nicer if this obj can be represented by a function

Reply:
Defining types for logs isn't very common probably since they can change iteratively so I'd leave it as it is unless you've a strong opinion.